packages/main/src/types/Ui5DomRef.ts

```diff
@@ -1,16 +1,52 @@
+import type { PropertyValue, SlotValue } from '@ui5/webcomponents-base/UI5ElementMetadata.js';
```

Comment:
This import might cause issues in bundlers which are interpreting the `exports` map in package.json.
I would recommend adding the `dist` path segment